2021-01-18 19:10:32,612 INFO    MainThread:3885368 [wandb_setup.py:_flush():70] setting env: {}
2021-01-18 19:10:32,612 INFO    MainThread:3885368 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-18 19:10:32,612 INFO    MainThread:3885368 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-18 19:10:32,612 INFO    MainThread:3885368 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedprox/wandb/run-20210118_191032-1tbvoinw/logs/debug.log
2021-01-18 19:10:32,613 INFO    MainThread:3885368 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedprox/wandb/run-20210118_191032-1tbvoinw/logs/debug-internal.log
2021-01-18 19:10:35,317 INFO    MainThread:3885368 [wandb_run.py:_console_start():1308] atexit reg
2021-01-18 19:10:35,319 INFO    MainThread:3885368 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-18 19:10:35,319 INFO    MainThread:3885368 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-18 19:10:35,319 INFO    MainThread:3885368 [redirect.py:install():205] install start
2021-01-18 19:10:35,320 INFO    MainThread:3885368 [redirect.py:install():220] install stop
2021-01-18 19:10:35,320 INFO    MainThread:3885368 [redirect.py:install():205] install start
2021-01-18 19:10:35,321 INFO    MainThread:3885368 [redirect.py:install():220] install stop
2021-01-18 19:10:35,321 INFO    MainThread:3885368 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-18 19:11:04,111 INFO    MainThread:3885368 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 0
2021-01-18 19:11:04,113 INFO    MainThread:3885368 [wandb_run.py:_restore():1250] restore
2021-01-18 19:11:04,115 INFO    MainThread:3885368 [redirect.py:uninstall():224] uninstall start
2021-01-18 19:11:04,116 INFO    MainThread:3885368 [redirect.py:_stop():279] _stop: stdout
2021-01-18 19:11:04,116 INFO    MainThread:3885368 [redirect.py:_stop():285] _stop closed: stdout
2021-01-18 19:11:04,117 INFO    stdout    :3885368 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-18 19:11:04,118 INFO    stdout    :3885368 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-18 19:11:04,118 INFO    MainThread:3885368 [redirect.py:_stop():291] _stop joined: stdout
2021-01-18 19:11:04,119 INFO    MainThread:3885368 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-18 19:11:04,119 INFO    MainThread:3885368 [redirect.py:uninstall():228] uninstall done
2021-01-18 19:11:04,119 INFO    MainThread:3885368 [redirect.py:uninstall():224] uninstall start
2021-01-18 19:11:04,119 INFO    MainThread:3885368 [redirect.py:_stop():279] _stop: stderr
2021-01-18 19:11:04,120 INFO    MainThread:3885368 [redirect.py:_stop():285] _stop closed: stderr
2021-01-18 19:11:04,120 INFO    stderr    :3885368 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-18 19:11:04,120 INFO    stderr    :3885368 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-18 19:11:04,120 INFO    MainThread:3885368 [redirect.py:_stop():291] _stop joined: stderr
2021-01-18 19:11:04,121 INFO    MainThread:3885368 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-18 19:11:04,121 INFO    MainThread:3885368 [redirect.py:uninstall():228] uninstall done
2021-01-18 19:11:04,646 INFO    MainThread:3885368 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "22a5ee6ad5f74dd386293062c8428fa4"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 635
      total_bytes: 635
    }
  }
}

2021-01-18 19:11:06,654 INFO    MainThread:3885368 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "31aedb9fce5f430f945e48dbad3a5f02"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 4569
      total_bytes: 4569
    }
  }
}

2021-01-18 19:11:08,192 INFO    MainThread:3885368 [wandb_run.py:_show_files():1614] logging synced files
2021-01-18 19:11:08,193 INFO    MainThread:3885368 [internal.py:handle_exit():139] Internal process exited
